Yona/Yona/AppDelegate.swift

```diff
@@ -41,6 +50,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
     }
 
+    //MARK: User Methods
+    func getScreenNameToDisplay() -> UINavigationController{
+        
+        
+        let viewName = getViewControllerToDisplay("ScreenToDisplay") as! String
+        var rootController : UIViewController
+        switch viewName {
+            
+        case "SMSValidation":
+            rootController = R.storyboard.sMSValidation.sMSValidationViewController! as SMSValidationViewController
+        case "Passcode":
+            rootController = R.storyboard.passcode.passcodeStoryboard! as SetPasscodeViewController
+        case "Login":
+            rootController = R.storyboard.login.loginStoryboard! as LoginViewController
+        default:
+            rootController = R.storyboard.welcome.welcomeStoryboard! as WelcomeViewController
+        }
+        
+        let a : UINavigationController? = UINavigationController(rootViewController: rootController)
+        return a!
```

Comment:
Forced unwrap the returning object? IS this safe? will crash...use a closure...also for style delete the space between "a" and ":" so "a:"

if let navigationControllerToDisplay = UINavigationController(rootViewController: rootController) {
     return navigationControllerToDisplay
} else {
}